dev/TeachingTip/TestUI/TeachingTipPage.xaml

```diff
@@ -571,5 +577,29 @@
             </StackPanel>
         </ScrollViewer>
 
+        <Grid Grid.Column="2">
+            <Button x:Name="targetButtonRightEdge" HorizontalAlignment="Right" AutomationProperties.Name="TargetButtonRightEdge" VerticalAlignment="Top" Click="OnShowButtonClickedRightEdge"
+                    Height="40" Width="100">Click to open TeachingTip
+                <Button.Resources>
+                    <muxc:TeachingTip x:Name="TeachingTipInResourcesOnEdge" 
```

Comment:
>TeachingTipInResourcesOnEdge [](start = 46, length = 28)

What is different about this teaching tip that enables the test? just that it is on  the right edge of the screen and more likely to move?